readme.md

```diff
@@ -62,6 +62,7 @@ I'm a big fan of [_Awesome_ selfhosted](https://github.com/awesome-selfhosted/aw
 ### PaaS
 
 - [Clever cloud](https://www.clever-cloud.com/) - Clever Cloud is an IT Automation platform. We manage all the ops work while you focus on your business value :fr:.
+- [Scalingo](https://www.scalingo.com/) - Scalingo is a high-availability platform as a service (PaaS) with an expert support. We remove the pain of handling servers, application stacks, software dependencies and databases. We host more than 50 different technologies and support 6 different databases.
```

Comment:
```suggestion
- [Scalingo](https://www.scalingo.com/) - Scalingo is a high-availability platform as a service (PaaS) with an expert support. We remove the pain of handling servers, application stacks, software dependencies and databases. We host more than 50 different technologies and support 6 different databases :fr:.
```

Reply:
Indeed! Sorry done :)

Reply:
@Ungredda sorry to be _psychorigid_ haha but can you put flag before the `.` to keep the same as other line :)

Reply:
Of course :) I think it should be done !